Description: Disable a check for the size of long double
 This check fails on armel, armhf, mips and mipsel,
 where sizeof(long double) == 8.
Author: Tobias Hansen <thansen@debian.org>

--- a/nall/stdint.hpp
+++ b/nall/stdint.hpp
@@ -47,7 +47,6 @@
 
 static_assert(sizeof(float)       >=  4, "float32_t is not of the correct size");
 static_assert(sizeof(double)      >=  8, "float64_t is not of the correct size");
-static_assert(sizeof(long double) >= 10, "float80_t is not of the correct size");
 
 using intmax = intmax_t;
 using intptr = intptr_t;
